examples/all_the_things.py

```diff
@@ -89,25 +90,28 @@ def dimensions(test):
 
 
 def attachments(test):
-  test.Attach('test_attachment', 'This is test attachment data.')
-  test.AttachFromFile('example_attachment.txt')
+  test.attach('test_attachment', 'This is test attachment data.')
```

Comment:
We need to start a changelog :) This is going to break all my team's testers twice over